scripts/upload_test_file.py

```diff
@@ -0,0 +1,52 @@
+import argparse
+from math import ceil
+from multiprocessing import Pool
+from botocore.exceptions import ClientError
+import logging
+import numpy as np
+
+from skyplane.obj_store.s3_interface import S3Interface
+
+if __name__ == "__main__":
+    """
+    Format:
+        -r : the region of the bucke. Defaults to us-east-2.
+        -mb : the total MB. Defaults to 4MB.
+        -n : the number of files. Defaults to 1.
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-r", help="The region of the bucke. Defaults to us-east-2.", default='us-east-2')
+    parser.add_argument("-mb", help = "Total MB size. Defaults to 4MB.", default=4)
+    parser.add_argument("-n", help="the number of files. Defaults to 1.", default=1)
+    args = parser.parse_args()
+    
+    bucket_name = input("Please input the name of bucket: ")
```

Comment:
Maybe make the bucket name an optional argument, and otherwise generate a random bucket name e.d. "skyplane-test-bucket-<RANDOM_HASH>"? Then if its a random bucket name we can also automatically cleanup at the end of the script by removing the bucket to avoid storage costs.